### ISSUE TITLE
Names of the items in the csv matrix output file

### ISSUE THREAD

**Issue by mancag** [jdemetra/jdemetra-app#348]
The names of the items in the csv matrix output file are not the same as in the »items« option. Consequently, »start« in the output file is used for both »span.start« and »espan.start« in the »items« option. This might be confusing, so I think it would be a good idea to have the same names in the output file as they are in the »items« option.

**Comment by palatej** [comment]
You are right. The only problem is that the "full names" are often quite long. It can be very annoying when you visualize the results in software like Excel. Perhaps we could add an option to choose short or long names for the output.
